manifests/40-webhook-configuration.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
We should probably increase the prefix number for this and the following files by 10. Or perhaps use `45-webhook-configuration.yaml` if changing the name of the other manifests could cause problems?

Reply:
yee, I want to avoid renaming, 45 will work for me